generation/new_client/README.md

```diff
@@ -136,8 +144,14 @@ create a pull request in the monorepo:
 ...
 Prepared new library in workspace/monorepo/java-apikeys
 Please create a pull request from that directory:
-  $ cd /usr/local/google/home/suztomo/google-cloud-java/generation/new_client/workspace/monorepo
-  $ gh pr create --title 'feat: [apikeys] new module for apikeys'
+  $ cd /usr/local/google/home/${USER}/google-cloud-java/generation/new_client/workspace/monorepo
+  $ gh pr create --title 'feat: [${API_SHORT_NAME}] new module for apikeys'
+```
+
+Install the Github CLI and login, if needed:
+```
+$ sudo apt-get install gh
+$ gh auth login
```

Comment:
Can you move this at the end of the "Prerequisites" section? Just before "## Run client generation script"

Reply:
Done.